doc/releases/changelog-dev.md

```diff
@@ -236,6 +236,10 @@
 
 * `pennylane.collections`, `pennylane.op_sum`, and `pennylane.utils.sparse_hamiltonian` are removed.
 
+* The `pennylane.transforms.qcut` module now uses `(op, id(op))` as nodes in directed multigraphs that are used within
+  the circuit cutting workflow instead of `op`. This change removes the dependency of the module on the hash of operators.
+  [(#4227)](https://github.com/PennyLaneAI/pennylane/pull/4227)
+
```

Comment:
Since we didn't have a deprecation cycle for this change, I'm not sure if this should be added to `deprecations.rst` or not.

Reply:
how does this change affect the user interface into qcut tools?

Reply:
Only if they directly try to manipulate the graph. The `tape_to_graph` and `fragment_graph` functions both return DAGs with operator nodes. Both those functions being public makes this change breaking

Reply:
in that case, I'd probably put it there bc it doesn't hurt. maybe @trbromley has a different opinion here?